scripts/requirements-build.txt

```
@@ -5,4 +5,4 @@ cryptography>=36.0.0
 imagesize>=1.2.0
 intelhex
 pylint
-zcbor
+zcbor>=0.8.0
```

Don't we limit this?
If 1.0.0 comes out and it changes API, it might break.

So should this be
```suggestion
zcbor>=0.8.1, == 0.8.*
```

And when 1.0 finally comes out, just (compatible release)
```
zbor ~= 1.0.0
```

---

I don't think `~=` is safe with `0.x.x` because it is `x.y.z ~= x.*.*` so this would be same as `0.*.*`

It depends on how Zcbor team keeps the API compatible. Hopefully 0.10, if ever published, is going to be compatible.

---

I think it only means 0.8.x: https://peps.python.org/pep-0440/#compatible-release

---

Btw. I don't mean that this has to be changed now. Many other rules in our `requirements.txt` are equally problematic as they use `>=`
I just wanted to point out that this might cause issues in future.